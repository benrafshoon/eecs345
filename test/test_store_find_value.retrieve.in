sleep 1
find_value 1000000000000000000000000000000000000000 1234567812345678123456781234567812345678
quit
