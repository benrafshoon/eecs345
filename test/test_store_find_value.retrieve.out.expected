[16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0] testing remote store and retrieve
